File: testing.py
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
from webdriver_manager.chrome import ChromeDriverManager

# driver = webdriver.Chrome(ChromeDriverManager().install())
# driver = webdriver.Chrome(executable_path="C:\\Users\JINGGA.BOT\node_modules\chromedriver\lib\chromedriver\chromedriver.exe", service_args=["--verbose", "--log-path=D:\\qc1.log"])
browser = webdriver.Chrome('D:\chromedriver\chromedriver.exe')
driver= webdriver.Chrome()

browser.get('http://www.youtube.com')
assert 'Youtube' in browser.title

elem = browser.find_element_by_name('search_query')  # Find the search box
elem.send_keys('Dr Phil' + Keys.RETURN)

browser.quit()

File: testing.js
// -------------------------------------
// |                                   |
// |        instalasi webdriver        |
// |                                   |
// -------------------------------------

// var webdriver = require('selenium-webdriver')
// const chrome = require('selenium-webdriver/chrome')
// const chromedriver = require('chromedriver')
// SeleniumServer = require("selenium-webdriver/remote").SeleniumServer;
const {Builder, By, Key, Util} = require ("selenium-webdriver");

// chrome.setDefaultService(
//     new chrome.ServiceBuilder(chromedriver.path).build())

//------------------------------------------------------------------------
//Testing
async function testing(){
    let driver = await new Builder().forBrowser("chrome").build();

    await driver.get("http://www.youtube.com");
    await driver.findElement(By.name("search_query")).sendKeys("Dr Phil", Key.RETURN);
}
testing();

// async () =>{
//     await driver.get('http://www.youtube.com')
//     const searchInput = await driver.findElement(webdriver.By.name('search_query'))

//     await searchInput.sendKeys('Dr Phil')

//     const button = await driver.findElement(webdriver.By.id('search-icon-legacy'))
//     const searchButton = button[1]

//     await searchButton.click()

//     await driver.sleep(2000)

//     driver.quit()
// }